modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/com.liferay.blogPosting.jar.portlet/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0

```

Comment:
I'm thinking if we are using a folder instead of a jar/war file, then maybe we can only have one new folder, then copy the original folders into it, at last to archieve a file, I think it should be work in general, but we'll see